cmd/containerd/command/service_windows.go

```diff
@@ -307,7 +307,7 @@ func launchService(s *server.Server, done chan struct{}) error {
 		done:    done,
 	}
 
-	interactive, err := svc.IsAnInteractiveSession() // nolint:staticcheck
+	interactive, err := svc.IsAnInteractiveSession() //nolint:staticcheck
```

Comment:
This one was added in https://github.com/containerd/containerd/commit/edc671d6a0192111e20578fc19531cb74fd8ce96 (https://github.com/containerd/containerd/pull/4626). I'll open a separate PR to replace it